### PR TITLE
feat(i18n): add Bahasa Indonesia (id-ID)

### PR DESCRIPTION
**Pull request by nyancodeid**
Hi, thanks for nice and helpful website. It's help us very much. Just tell everybody to scan and boom, they connected. 

This PR adds the Bahasa Indonesia (id-ID) translation. 

Thank you :+1:

### CODE REVIEW

**Comment by bndw** [comment]
Hey @nyancodeid thank you for the translation! Would you mind rebasing master and I'd be happy to merge!

File: src/i18n.js [hunk 1]
```diff
@@ -208,6 +208,31 @@ const resources = {
       select: 'Вибір мови',
     },
   },
+  'id-ID': {
+    translation: {
+      title: 'kartu WiFi',
```
Comment (bndw) [inline src/i18n.js:213]: Should Kartu be capitalized?

Comment (nyancodeid) [reply src/i18n.js:213]: Yes, it should be capitalized. I'll fix it tomorrow.

**Comment by nyancodeid** [comment]
Hi @bndw , it's should be okay now. But, It's my first time doing rebase hopefully it's correct. Thank you.

**Comment by bndw** [comment]
Hi @nyancodeid , it looks like the rebase pulled in some extra stuff. I've grabbed your changes and opened a new pull request https://github.com/bndw/wifi-card/pull/115 -- Would you mind verifying it looks good and we can merge that one instead?

cc: @riipandi

**Pull request closed without merging**